config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -138,6 +138,8 @@ endif # SOC_SERIES_NRF53X
 
 endif # CHIP_DFU_OVER_BT_SMP
 
+if CHIP_WIFI
```

Comment:
Isn't it enough that each of these configs `depends on` CHIP_WIFI?

Reply:
I would bet there is a difference, so settings some config that has unmet `CHIP_WIFI` dependency through `depends on` will only generate a warning that final value is different than desired, and settings some config that has unmet `if` statement before declaration will result in generation failure due to undefined symbol. Interestingly there seem to be no difference, so I was wrong. I will remove this change.